dune-project

```diff
@@ -1,3 +1,3 @@
-(lang dune 2.5)
-(using coq 0.2)
+(lang dune 3.3)
+(using coq 0.4)
```

Comment:
Why not take the opportunity to use 3.6 and 0.7? Bignums is a library for effective computations, having it incompatible with coq-native doesn't make much sense.

Reply:
I am weary of using too high of a Dune version as to make this fail to build in Coq's CI. Of course, we should really bump the Dune version there, but I haven't really been motivated to do so yet.

Reply:
Coq CI doesn't use dune for bignums

Reply:
OK thanks, I've updated it now.

Reply:
No

https://github.com/ocaml/dune/blob/a5fd306a4e24c7bd67330b411b3e5662b7e03039/src/dune_rules/coq/coq_stanza.ml#L4-L13